examples/decoupled/repeat_model.py

```diff
@@ -135,7 +135,16 @@ def execute(self, requests):
             raise pb_utils.TritonModelException("unsupported batch size " +
                                                 len(requests))
 
-        # Start a separate thread to send the responses for the request.
+        # Start a separate thread to send the responses for the request. A model using
+        # decoupled transaction policy is not required to send all responses for the
+        # current request before returning from the execute. The sending the responses
+        # is delegated to this thread.
+        # This means the model can process multiple requests at the same time in
+        # the same model instance, as it defers responding to requests in an
+        # earlier invocation until after responding to requests from a later request.
+        # Even though this demonstrates the flexibility the decoupled API,
+        # it can lead to many requests into pipeline whose responses are being
+        # processed. A model developer should be mindful of this.
```

Comment:
Can you clarify more on why the user should be mindful of this? Are you saying that there could be many requests that Triton core needs to keep track of or you are saying that there could be many responses in the queue that Python backend is copying/retrieving from shared memory?